orchagent/portsorch.cpp

```diff
@@ -534,6 +603,10 @@ void PortsOrch::doPortTask(Consumer &consumer)
                 if (fvField(i) == "mtu")
                     mtu = (uint32_t)stoul(fvValue(i));
 
+                /* Set port pvid */
+                if (fvField(i) == "pvid")
+                    pvid = (uint32_t)stoul(fvValue(i));
```

Comment:
why do we need this? we can keep the assumption that the untagged mode vlan is the pvid of the port.

Reply:
The original design is to support pvid setting on port and lag from admin interface,  as provided at SAI api.  Since it is causing a lot of confusion and no actual use case for it yet,  removing support for explicit pvid configuration.